src/test/java/nl/hsac/fitnesse/fixture/util/JsonHelperTest.java

```diff
@@ -89,20 +90,20 @@ public void testSortingOnNumber() {
                 "  ]\n" +
                 "}", "$.parameters", "$.price");
 
-        assertEquals("{\n" +
-                        "    \"extraKey\": 2,\n" +
-                        "    \"parameters\": [\n" +
-                        "        {\n" +
-                        "            \"price\": 8.95,\n" +
-                        "            \"category\": \"reference\"\n" +
-                        "        },\n" +
-                        "        {\n" +
-                        "            \"price\": 18.95,\n" +
-                        "            \"category\": \"areference\"\n" +
-                        "        }\n" +
-                        "    ]\n" +
+        JSONAssert.assertEquals("{\n" +
```

Comment:
Why do we need to use JSONAssert here, and especially with `strict=false`? I thought the whole point of this PR was that the JSON was completely stable?

Reply:
@fhoeben JSONAssert has two strict modes. If you set the strict mode to `false`,  then this means that compare the two jsons but the order doesn't matter. If it's true, then it will compare with two jsons while checking for the order. 

For the tests `SortingOnNumber` and `testSortingOnString`, the tests are coming out of order each time. When debugging there were changes to be done but to avoid a substantial amount of change. I decided to introduce JSONAssert which does assertEquals but doesn't change the order but everything else is checked. 

E.g. of strict mode = false
Json 1: 
``` json
{
  "firstName": "Billy",
  "lastName": "Sandy"
} 
```

Json 2 (different order):
``` json
{
  "lastName": "Sandy",
  "firstName": "Billy"
} 
```

JSONAssert will compare these two json but the order won't matter as long as long as everything else is the same.
Here's the link to the library: https://github.com/skyscreamer/JSONassert.